### DISCUSSION TITLE
503 Service Unavailable when editing an uploaded image or document

### DISCUSSION THREAD

**Discussion by mbaderweb** [sulu/sulu#6889] [Q&A] [answered]
| Q | A
| --- | ---
| 2.4.0 | Sulu
| 8.1 | PHP

# Actual Behavior

When i edit an image or document in the media section in admin, it try's to navigate to the file view but then when the loader appears nothing happens. I checked the specific request in the background and the media request says 503 Service Unavailable.

When I clear the cache, it works for one time, then the error occurs again.

I deployed the sulu cms to anonther managed server on a different hoster and all works fine. 
I checked the php_info on both systems and I realize that opcache on the problematic system is missing.

The problem occurs only in prod mode.

**Comment by mbaderweb** [comment]
There are no accorded error messages in the log files..nothing

**Comment by alexander-schranz** [comment, marked as answer]
@mbaderweb sorry I think without any logs or stack trace of an error its not possible to help here a lot. I think you need to try to reproduce the issue. Example you can run the application locally with `APP_ENV=prod` and check if its a problem with the `APP_ENV` or with your server. If its only your server it can also be that you don't have enough resources.

But no logs are always strange. Please check that your log directories are writeable by the your webserver or php-fpm user else it can not write any logs. Try to create a simple throw new \RuntimeException error in an example controller to check that the log dirs are correctly writeable and errors are logged.

As you are on a older version (2.4.0) I can at current state just recommend you update your project one of the latest supported versions (2.4.6 and 2.5.2).

Please keep in mind the `Media` View generate the thumbnails. And if you are not using the recommended VHOST of sulu:

 - https://docs.sulu.io/en/latest/cookbook/web-server/apache.html
 - https://docs.sulu.io/en/latest/cookbook/web-server/nginx.html
 
It can end that every opening will regenerate the thumbnails and your server has not enough resources to generate that big size of images. So check your vhost only the first request to a image thumbnail should be slow and additional request should directly return the image without calling PHP process.

**Comment by mbaderweb** [comment]
@alexander-schranz thank you for your ultra fast answer, that is amazing!! 
I also think, this is an resource issue.
I will try your recommendations
Thanks!!

**Reply by mbaderweb** [reply]
Changing to php8.0 fixed that issue...
